presto-docs/src/main/sphinx/release/release-0.238.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.238
+=============
+
+**Highlights**
+==============
+* Fix case sensitivity issue in SQL function parameters. Function parameters should be treated case-insensitive.
+* Add support to create external function.
```

Comment:
Just external function is not. Lets add a reference to a Github issue to explain what it does. @rongrong

Reply:
linking issue.